### DISCUSSION TITLE
Limiting data to buildings with names

### DISCUSSION THREAD

**Discussion by ResearchICTSolutions** [OvertureMaps/data#46] [Discussion]
I want to show only those buildings that have a names field that is not empty. This is the code I've been using: 
SELECT
    CAST(names AS JSON),
    geometry -- WKB
FROM
    buildings
WHERE
    bbox.minX > 11.5
    AND bbox.maxX < 25.5
    AND bbox.minY > -29.0
    AND bbox.maxY < -17.0
    AND names IS NOT NULL
    AND CAST(names AS VARCHAR) <> '{}'

However, this query fails. Here are my questions: 
What is the SQL dialect that Overture uses? 
How can I modify my query so that I get results that has data in the names field?

**Comment by jenningsanderson** [comment]
Find more examples here https://docs.overturemaps.org/getting-data/ — but with the latest Overture data, you can query where `names.primary is not null` to obtain buildings with names.
